<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PingMainWindow</class>
 <widget class="QMainWindow" name="PingMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ping Me!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="urlLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="urlLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="startPushButton">
        <property name="text">
         <string>Start Ping!</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/add.png</normaloff>images/add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopPushButton">
        <property name="text">
         <string>Stop Ping </string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/stop.png</normaloff>images/stop.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
